src/content/api/media-uploads.mdoc

```diff
@@ -44,3 +44,69 @@ When the Media Upload was updated.
 The User or API Key that updated the Media Upload.
 {% /property %}
 {% /properties %}
+
+{% endpoint method="POST" path="/api/media-uploads" %}
+## Create a presigned URL for Media Upload EXPERIMENTAL
+{% row %}
+{% column %}
+### Required Fields
```

Comment:
@AndyClifford I think this should now say `Attributes` and the fields should have the required badge on them.
See design: https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B/Centrapay-Docs-(Tailwind)?type=design&node-id=1691-2617&mode=design&t=50p3UEnXzuaCVT5Q-4